CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ MESSAGE(STATUS "nanoflann version: ${NANOFLANN_VERSION_MAJOR}.${NANOFLANN_VERSIO
 file(WRITE "${nanoflann_BINARY_DIR}/version" "${NANOFLANN_VERSION_MAJOR}.${NANOFLANN_VERSION_MINOR}.${NANOFLANN_VERSION_PATCH}")
 
 # Compiler options:
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
This will only work for GCC and would be a warning/error for MSVC, for example... 

Please, change it to something like [this](https://github.com/MRPT/mrpt/blob/master/CMakeLists.txt#L27) ;-)

Reply:
Yes, I am aware of this very nice feature. However, as long as Ubuntu 14.04 with CMake 2.8.12 is supported, we cannot use this features as it was introduced in CMake 3.1+ (see https://cmake.org/cmake/help/v3.1/release/3.1.0.html#properties).
If I remember correctly from other projects, MSVC defaults to C++11 or at least automatically uses it. So only applying the flag for GCC should work, I think.

Reply:
Fixed this in the latest commit. Maybe adding Appveyor as a Windows CI would help here to catch such things. Anyways, thanks for pointing me to this issue.

Reply:
You're right, it's ok to support Ubuntu 14.04 until its EOL.